busybox/busybox.py

```diff
@@ -9,7 +9,7 @@
 
 from trunner.ctx import TestContext
 from trunner.dut import Dut
-from trunner.types import TestResult, TestSubResult, Status
+from trunner.types import TestResult, Status
 
 
 def harness(dut: Dut, ctx: TestContext, result: TestResult):
```

Comment:
Please provide a few characteristic cases from both mbedtls and busybox output in those harness files to make it more understandable and prepare some data for tests (testing the harnesses) as @nalajcie suggested before.

```suggestion
# exemplary output from the busybox testsuite:
# ...

def harness(dut: Dut, ctx: TestContext, result: TestResult):
```